crates/ruff_workspace/src/options.rs

```diff
@@ -1,3 +1,5 @@
+//! Find configurations provided by tools like flake8, pydocsyle and isort.
```

Comment:
This isn't quite right. These are the raw options that are read from `pyproject.toml` or `ruff.toml`. They're defined within Ruff and passed to the various packages that rely on them (e.g., our isort implementation).

We then have `Configuration` which is an intermediate representation, and `Settings` which is the final internal representation that _actually_ powers Ruff. But `Options` need to match the TOML schema _exactly_.

Reply:
Oh lol thanks i was wondering how weird what do you need isort configs for. will rephrase